platform/broadcom/sonic-platform-modules-nexthop/nh-4010/service/pddf-platform-init.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
This file can be a soft-link to platform/pddf/i2c/service/oddf-platform-init.service file to avoid multiple copies of the same file in the repo.